src/includes/Exports-Include.hs

```diff
@@ -96,12 +128,15 @@
   , scrollPageUpCode
   , scrollPageDownCode
 
-    -- * Select Graphic Rendition mode: colors and other whizzy stuff
+    -- * Select Graphic Rendition mode: colors and other whizzy stuff.
+    -- | These code sequences are part of ECMA-48 standard and are portable.
   , setSGR
   , hSetSGR
   , setSGRCode
 
     -- * Cursor visibilty changes
+    -- | These code sequences are not part of ECMA-48 standard; they are popular,
+    -- but non-portable extensions. E. g., ANSI.SYS does not support them.
```

Comment:
Windows does support these functions in its native terminals, so is not a good example for exceptions to portability.